src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -150,8 +154,11 @@ public Hardware() {
         if (INDEX_ENABLED) {
             ingestIndexMotor = new WPI_TalonFX(INDEX_INGEST_MOTOR);
             feederIndexMotor = new WPI_TalonFX(INDEX_FEEDER_MOTOR);
-            ingestIndexColorSensor = new MultiplexedColorSensor(INDEX_INGEST_SENSOR);
-            feederIndexColorSensor = new MultiplexedColorSensor(INDEX_FEEDER_SENSOR);
+             ingestProximity  = new DigitalInput(0);
```

Comment:
There is an extra space at the beginning of lines 157, 158.
Please run 
./gradlew :spotlessApply

to fix formatting issues and push the changes up again.

Reply:
done